### ISSUE TITLE
USNM: National Museum of Natural History

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#5994]
## Instructions

This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions.

Reviewing documentation before proceeding will result in a more enjoyable experience.

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step.

All [USNM: National Museum of Natural History](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type#usnm__national_museum_of_natural_history) should be replaced with other ID type = other identifier and issued by agent [Smithsonian National Museum of Natural History](https://arctos.database.museum/agent/21281779)

**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos. 

**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable.

**Context**: Describe why this new value is necessary and existing values are not.

**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure.

**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm

**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking.

**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions. 

**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

## Approval

All of the following must be checked before this may proceed.

The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality).

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc. 

1. Can a suitable solution be found here? If not, proceed to (2)
2. Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)
3. Take the discussion to a monthly Arctos Working Group meeting for final resolution.

## Implementation

Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made.

Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel. 

Make changes as described above. Ensure the URL of this Issue is included in the definition. 

Close this Issue.

**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result.

## Special Exemptions

In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases.

1. Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step.
2. The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology.

**Comment by dustymc** [comment]
I will plan on proceeding with this about 2023-03-28 if there are no objections.

I will proceed immediately upon approval of each of the involved collections.

Data:
[temp_usnm_national_museum_of_natural_history.csv.zip](https://github.com/ArctosDB/arctos/files/10973257/temp_usnm_national_museum_of_natural_history.csv.zip)


Summary:
|guid_prefix|numrecs|
|-----------|--------|
|ALMNH:Inv|7|
|ALMNH:Paleo|1|
|CHAS:Bird|36|
|CHAS:Mamm|6|
|CRCM:Bird|11|
|DMNS:Bird|9|
|DMNS:Mamm|5|
|MLZ:Bird|68|
|MSB:Bird|33|
|MSB:Fish|1|
|MSB:Host|25|
|MSB:Para|45|
|MVZ:Bird|36|
|MVZ:Herp|6|
|MVZ:Mamm|1|
|NMMNH:Paleo|1|
|UAM:Art|9|
|UAM:Herp|1|
|UAM:Inv|11|
|UAM:Mamm|25|
|UAMObs:Ento|33|
|UCM:Egg|3|
|UCM:Fish|72|
|UCM:Herp|2|
|UCM:Mamm|9|
|USNPC:Para|1|
|UTEP:ES|24|
|UTEP:Inv|5|


Users:
@ebraker
@msbparasites
@Nicole-Ridgwell-NMMNHS
@mlbowser
@mkoo
@jessicatir
@campmlc
@ccicero
@catherpes
@amgunderson
@atrox10
@DerekSikes
@marecaguthrie
@mvzhuang
@cjconroy
@wellerjes
@kmkocot
@jandreslopez
@droberts49
@aklompma
@acdoll
@jrdemboski
@kyndallh


See also https://github.com/ArctosDB/arctos/issues/5771

**Comment by campmlc** [comment]
Not OK for MSB:Host and MSB:Para collections until further discussion
regarding impact on relationships.

**Comment by dustymc** [comment]
>impact on relationships

If you have a concern please open an Issue to it can be addressed.

**Comment by campmlc** [comment]
This is the issue I'm using for now. When I can stop reviewing the endless flow of new issues, I can consider taking a breath to create one.

**Comment by campmlc** [comment]
https://github.com/ArctosDB/arctos/issues/6004

**Comment by wellerjes** [comment]
Reviewed; okay to proceed for CHAS

**Comment by ebraker** [comment]
Ok for UCM

**Comment by Jegelewicz** [comment]
I think we can do more with this one.

I just worked on UTEP:Inv and I can find all of the related identifiers on the [Smithsonian's Invertebrate Zoology](https://arctos.database.museum/agent/21347813) database. This brings all of those records one step closer to an actual link with USNM:IZ.

![image](https://user-images.githubusercontent.com/5725767/225681677-1220b3a4-a111-4215-b266-190589cf52dc.png)

![image](https://user-images.githubusercontent.com/5725767/225681912-4f863405-0663-41f8-8049-56879b3b3a8e.png)

**Comment by Jegelewicz** [comment]
NMMNH:Paleo 

https://arctos.database.museum/guid/NMMNH:Paleo:43252

@Nicole-Ridgwell-NMMNHS I can't find the corresponding record on the USNM Paleo site - https://collections.nmnh.si.edu/search/paleo/

**Comment by Jegelewicz** [comment]
UTEP:ES

@mvzhuang I also could not find the UTEP USNM identifiers in their paleo site search - https://collections.nmnh.si.edu/search/paleo/

here is the list.
https://arctos.database.museum/search.cfm?guid_prefix=UTEP%3AES&oidtype=USNM%3A%20National%20Museum%20of%20Natural%20History

**Comment by campmlc** [comment]
That's great - I'd like to see if we can do that for MSB links. And again,
if we can narrow down to an actual institutional catalog number at USNM,
then that is the other ID type we would request.

**Comment by Jegelewicz** [comment]
>  variously-formatted possibly-random IDs or looking up ARKs manually or ??

All of this comes straight from GRSciColl (not so useless now?)

[Smithsonian Institution Page](https://www.gbif.org/grscicoll/institution/586ee56e-b0fe-4dff-b7f9-aeb104f3308a)
![image](https://user-images.githubusercontent.com/5725767/225690820-4884f160-c7e8-48ae-83c8-55d0bef982d5.png)

Contains links to all of their collections
![image](https://user-images.githubusercontent.com/5725767/225691016-d4cf228e-45cd-47f9-94c0-d374326774ef.png)

Collection pages like [Invert Zoo](https://www.gbif.org/grscicoll/collection/0174f5b3-da29-4967-b8dc-ce75ed53e35d)
![image](https://user-images.githubusercontent.com/5725767/225691269-7a740f94-d7ea-4508-ac27-6974cef4e962.png)

Contain links to their [database search page](https://collections.nmnh.si.edu/search/iz/)
![image](https://user-images.githubusercontent.com/5725767/225691343-074ea22f-34fc-47cb-9b54-76cbfe633a06.png)

**Comment by dustymc** [comment]
Oh - I thought perhaps you were somehow getting from the shady IDs to ...

<img width="450" alt="Screenshot 2023-03-16 at 9 45 08 AM" src="https://user-images.githubusercontent.com/5720791/225691967-551fa764-6c15-47d9-908a-94603db3021c.png">

...the really great IDs.

**Comment by Jegelewicz** [comment]
Yeah, but that is a lot more work. Maybe we can figure out some way to go and get them all someday - but that seems to be a fundable project? Doing real digital extended specimen stuff?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Ours is actually a USNM locality ID, I've changed the identifier to locality ID issued by USNM (specimen is ungeoreferenced and in a general catch-all locality for this particular adopted collection, otherwise I would have added the ID to the locality).

**Comment by jessicatir** [comment]
OK for CRCM:Bird.

**Comment by Jegelewicz** [comment]
> I thought perhaps you were somehow getting from the shady IDs to ...
> 
> Screenshot 2023-03-16 at 9 45 08 AM
> 
> ...the really great IDs.

But the problem is @campmlc added them as complete links and we are adding on a base url, so they just 404. See https://arctos.database.museum/guid/MSB:Host:11676 where I have also added an ARK that functions properly.

There is definitely opportunity to make all of these ACTUAL links, but we have two ways of doing this and to any user unaware, this can result in malformed links.

I am happy to do some cleanup to get these working - and even to get the ARKs for the other stuff that I can, it will just take a while.

**Comment by Jegelewicz** [comment]
> EZID is an issuer/resolver, not an ID type!)

It is BOTH - the concept that I think we have been trying to get across for a while?

**Comment by dustymc** [comment]
>added them as complete links and we are adding on a base url,

Blargh, why again is https://github.com/ArctosDB/arctos/issues/6163 not my only priority?!


And ARK is another place where that plus the issuer really shines - nobody cares that it's an ARK, we care that is DOES STUFF and that it's "theirs." Old model completely fails at that, current handles it elegantly. But here we are for some reasons...


>It is BOTH

No (but I think what you're trying to say isn't wrong). EZID is an organization. They issue ARKs (which technically are part of the URL). They also resolve ARKs. You can also issue ARKs if you want, and you can resolve ARKs if you want, and EZID can resolve your ARKS (even though the URL might suggest otherwise) and vice-versa. It's complicated, even before we [arbitrarily rip them apart](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_other_id_type#ark) for Arctos (in an effort to confuse everyone? It works!).

It ain't gotta be that way, and nobody needs to understand any of that. Just paste the thing that DOES STUFF in, save,  and be done with it. Why is that contentious?!

>also added an ARK

FWIW I'd call that one an identifier (if I could - I can't even find that issue!) - it's clearly not their catalog number, see above re: ARK type, there's no reason to complicate this.

**Comment by Jegelewicz** [comment]
Unable to find the UTEP:ES numbers in any of the Smithsonian databases, so changed them to issued by [Smithsonian National Museum of Natural History](https://arctos.database.museum/agents.cfm?agent_id=21281779)

**Comment by campmlc** [comment]
I am concerned that someone thought these were USNM numbers, and from my experience, the fact they aren't in the USNM digital catalog does not mean they aren't. These could just not be cataloged yet, or they are older numbers that the Smithsonian has replaced with newer ones. When I choose a USNM identifier for a record, I am explicitly choosing that number because I have "USNM" written on a tag. There is a big difference between an explicit "USNM" identifier and "issued by the Smithsonian", which is a giant institution with two different administrations. Again, I am very, very concerned about the loss of data that is going on here with these conversions. **Please stop** until we can have a committee to discuss.

**Comment by campmlc** [comment]
You do not have MSB permission to make any changes to MSB records.

**Comment by Jegelewicz** [comment]
Ah-Ha! I am pretty sure those UTEP:ES numbers are locality identifiers! I just sent an email to Art to confirm.

**Comment by Jegelewicz** [comment]
FWIW - I did not "say they are not USNM numbers" - they are all prefixed with USNM and the issuer is **[Smithsonian Museum of Natural History](https://arctos.database.museum/agents.cfm?agent_id=21281779)** which IS USNM. I even took the extra time to add the old USNM:National Museum of Natural History identifier to the remarks for the updated identifiers.

NOTHING has been lost and I stand by my work as making things better and more discoverable.

**Comment by KyndallH** [comment]
UAM:Mamm is taking care of the 25 they have on the list.

**Comment by Jegelewicz** [comment]
@KyndallH you might want to check out the additional conversation at https://github.com/ArctosDB/arctos/issues/6356

If you want to take time to look up the ARK's you could make some nice links...

**Comment by campmlc** [comment]
I do not question that some of this is valuable, or that it isnt being done for a good reason.  I am asking, yet again, for any changes to MSB records to wait until MSB personnel are available to evaluate. We are all getting ready for a massive field season, and no one is available to look at what is being done or proposed.

**Comment by campmlc** [comment]
@mkoo

**Comment by cjconroy** [comment]
MVZ mammals has only one record related to this. I pulled the catalog card. It is a tissue sample-only that came to us via the NIH. However, the card reads that it originated with the National Zoological Park, #39700. 
I changed this other id to an other ID issued by the Smithsonian National Zoological Park and deleted the USNM reference. 

Does this seem appropriate since we know it is from the zoo?

That agent is also used by MSB, DMNS:Mamm, UWBM:Mamm and UWZM:Bird in different ways.

**Comment by Jegelewicz** [comment]
@cjconroy that is the power of the issued by agent - The National Zoo can be a collector, determiner of many things and can issue identifiers for stuff! I think what you did makes perfect sense.

**Comment by dustymc** [comment]
Yes agreed - it's overwhelmingly obvious that the old types are being overloaded, and much (all??) of what they hold isn't what seems to be the general expectation, catalog numbers (that aren't attributed to collections for some reason...). There's no possible way we could have managed that under the old system, but it's trivial under the new.

* Institution
    * Division
        * Subdivision
            * Department
                * Subdepartment
                     * Project
                     
with Project issuing an identifier or two isn't difficult to set up, isn't difficult to use, and I think isn't difficult to understand.

Or, skipping the cleanup and flopping the types to "institution" is exactly what's being said (however incorrectly...) now, it just brings the data into a system where they *can* be cleaned up.
